src/templates/ar/2019/base.html

```diff
@@ -3,79 +3,128 @@
 {% block dir_attribute %}dir="rtl"{% endblock %}
 
 {% block styles %}
-  <link rel="stylesheet" href="/static/css/2019.css">
+  <link rel="stylesheet" href="/static/css/2019.css?v=2">
 {% endblock %}
 
 {% block content%}
+  <div id="skiptocontent"><a href="#maincontent">Skip navigation</a></div>
   {% block header %}
   <header class="alt-bg">
-    <div class="top-header">
-      <a class="navigation-logo" href="{{ url_for('home', year=year, lang=lang) }}"><img src="/static/images/logo.svg" alt="Web Almanac by HTTP Archive" height="39" width="202"></a>
-      <nav title="Desktop header menu">
-        <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}">Table of Contents</a>
-        <a href="{{ url_for('contributors', year=year, lang=lang) }}">Contributors</a>
-        <a href="{{ url_for('methodology', year=year, lang=lang) }}">Methodology</a>
-        {% if supported_years | length > 1 %}
-          {{ year_switcher() }}
-        {% endif %}
-      </nav>
-      <div class="cta">
+    <div class="container">
+      <div class="top-header">
+        <a class="navigation-logo" href="{{ url_for('home', year=year, lang=lang) }}"><img src="/static/images/logo.svg" alt="Web Almanac by HTTP Archive" height="39" width="202"></a>
+        <nav title="Page navigation">
+          <ul>
+            <li><a href="{{ url_for('table_of_contents', year=year, lang=lang) }}">Table of Contents</a></li>
+            <li><a href="{{ url_for('contributors', year=year, lang=lang) }}">Contributors</a></li>
+            <li><a href="{{ url_for('methodology', year=year, lang=lang) }}">Methodology</a></li>
+            {% if supported_years | length > 1 %}
+            <li>
+              {{ year_switcher() }}
+            </li>
+            {% endif %}
+            {% if supported_languages | length > 1%}
+            <li>
+              {{ language_switcher() }}
+            </li>
+            {% endif %}
+          </ul>
+        </nav>
         {% if request.endpoint == 'home' %}
+        <div class="cta">
           <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}" class="alt btn">
             Start Exploring
           </a>
+        </div>
         {% endif %}
-        {{ language_switcher() }}
+        <nav title="Page menu" id="Menu" aria-labelledby="MenuBtn">
```

Comment:
This is introducing a new naming convention. Throughout the site we use lowercase and hyphens.
```suggestion
        <nav title="Page menu" id="menu" aria-labelledby="menu-btn">
```

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/666/commits/1d5172131674bfb0a4e5f21ed6431d4a09e56eef